docs/CONTRIBUTING.md

```diff
@@ -1,38 +1,136 @@
 # Contribution guidelines 
 
-## How to contribute to project? 
-All contributions to this project are welcome. To propose changes, we encourage contributors to:
-1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this project on GitHub
-1. Create a new branch, and
-1. Propose their changes by opening a [new pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests). 
+If you’re reading this section, you are probably interested in contributing to the Canonical GTFS Schedule Validator. First, thank you for your interest, and welcome to the community! This document describes the different ways you can contribute to this project, and what the processes look like. 
```

Comment:
nit: maybe introduce a blank newline after this paragraph so it renders as a separate block?  Right now, it gets merged with the following paragraph when rendered from Markdown.

Reply:
Good catch :)